jnigen/lib/src/bindings/dart_generator.dart

```diff
@@ -46,6 +46,7 @@ const _deleteInstruction =
     '  /// The returned object must be deleted after use, '
     'by calling the `delete` method.';
 
+// Helper methods
```

Comment:
nit: The comment doesn't add anything, unnamed extensions are scoped to the file. Maybe just remove the comment.